### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: No need to run git clone - you can copy the repo with `ADD .`. Also, is it possible to use a package stage to reduce image size?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/sonic:latest
```
Comment (xansec) [inline Mayhemfile]: Tests has been renamed to testsuite. You may omit the image field.

**Review by xansec — CHANGES_REQUESTED** [review]
Is it necessary to upload the whole corpus? Usually a seed of 1 or 2 files is sufficient. Please make the requested changes.

**Comment by capuanob** [comment]
@xansec I have made the requested changes, looking forward to your feedback!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**